tests/SparqlLuaTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use MediaWiki\Http\HttpRequestFactory;
+use PHPUnit\Framework\TestCase;
+use ProfessionalWiki\SPARQL\SparqlLuaQueryRunner;
+
+class SparqlLuaTest extends TestCase {
+
+	/**
+	 * @covers SparqlLuaQueryRunner::runQuery
+	 */
+	public function testRunQuery() {
+		global $wgSPARQLEndpoint;
```

Comment:
Unit tests should not use global state, and since you are mocking the request handling anyway, you can just use a hardcoded dummy value like `https://example.com/whatever`